src/keas/kmi/facility.py

```diff
@@ -100,7 +106,9 @@ def encrypt_file(self, key, fsrc, fdst, chunksize=24*1024):
         encryptionKey = self._bytesToKey(self.getEncryptionKey(key))
 
         # 2. Create a random initialization vector
-        iv = ''.join(chr(random.randint(0, 0xFF)) for i in range(16))
+        # bytes(bytearray(generator)) is needed for Python 2,
+        # with Python 3 bytes(generator) works
+        iv = bytes(bytearray((random.randint(0, 0xFF)) for i in range(16)))
```

Comment:
I'm not a cryptographer, but I think it's a good idea to use a strong RNG for IVs --> use `os.urandom(16)`?  That would also make the code simpler.